plugins/inputs/sqlserver/README.md

```diff
@@ -172,9 +172,34 @@ GO
   ## - VolumeSpace
   ## - PerformanceMetrics
 
+```
+
+### Support for Azure Active Directory (AAD) authentication using [Managed Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
+
+Azure SQL Database supports 2 main methods of authentication: [SQL authentication and AAD authentication](https://docs.microsoft.com/en-us/azure/azure-sql/database/security-overview#authentication). The recommended practice is to [use AAD authentication when possible](https://docs.microsoft.com/en-us/azure/azure-sql/database/authentication-aad-overview).
 
+AAD is a more modern authentication protocol, allows for easier credential/role management, and can eliminate the need to include passwords in a connection string.
 
+To enable support for AAD authentication, we leverage the existing AAD authentication support in the [SQL Server driver for Go](https://github.com/denisenkom/go-mssqldb#azure-active-directory-authentication---preview)
 
+#### How to use AAD Auth with MSI
+
+- Configure managed identity for Azure resources on the Monitoring VM (the VM that'd connect to the SQL server/database) [using the Azure portal](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-portal-windows-vm).
```

Comment:
would it work with both user and system identity?

Reply:
User identity won't work here, since we retrieve token based on managed identity.

Reply:
User Identity is also a managed identity- https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-to-manage-ua-identity-portal#:~:text=1%20Sign%20in%20to%20the%20Azure%20portal%20using,of%20a%20user-assigned%20managed%20identity%20click%20its%20name. Can you help check? If it does not work, documentation might have to be explicit saying system-assigned identity.

Reply:
Yes, this is not meant for a user-assigned managed identity, I'll update the verbiage to call out that only a system-assigned managed identity should be created for this case.